string-manipulation-3/src/is-anagram.js

```diff
@@ -11,9 +12,22 @@ use a for loop to loop through combined string
 return true
 */
 
+function removeSpaces(string) {
+  var result = '';
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] !== ' ') {
+      result += string[i];
+    }
+  }
+  return result;
+}
+
 function isAnagram(firstString, secondString) {
   var toy = firstString + secondString;
   var count = 0;
+  if (removeSpaces(firstString).length !== removeSpaces(secondString).length) {
+    return false;
+  }
```

Comment:
This doesn't address the source of the issue I showcased in my last review, the arguments I used in the test I showed you did not include spaces.

Reply:
Hello Cody, I am a little confused. I thought by adding in a condition to check if the length of the first string is equal to the length of the second string would address the concern in your previous example. As in, if the lengths of the strings are not equal, then they can not be an anagram for each other

I added the removed spaces function so it could work with the other test cases that did include spaces.

Reply:
Disregard previous feedback here, this will work as intended.